dashboard/src/utils/axiosInstance.js

```diff
@@ -1,16 +1,15 @@
 import axios from "axios";
-import Cookies from "js-cookie";
 import { showSessionExpired } from "actions/toastActions";
 import store from "store/store";
 
-const { dispatch } = store;
+const { dispatch, getState } = store;
 
 const axiosInstance = axios.create({ responseType: "json" });
 
 axiosInstance.interceptors.request.use(async (req) => {
-  const token = Cookies.get("token");
-  if (token) {
-    req.headers.Authorization = `Bearer ${token}`;
+  const keycloak = getState().apiEndpoint.keycloak;
+  if (keycloak.authenticated) {
```

Comment:
You're sure that `getState().apiEndpoint` always has a `keycloak` property and that `keycloak` always has an `authenticated` property?  This is one of the places that I _would_ recommend using `?.`, because, when you get to the `if` at line 11, if any of the properties aren't set, then we'll do the right thing (I think), and that's probably better than getting an exception.